src/main/java/com/jcraft/jsch/KnownHosts.java

```diff
@@ -29,24 +29,30 @@ LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
 
 package com.jcraft.jsch;
 
-import java.io.*;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Vector;
 
 class KnownHosts implements HostKeyRepository{
-  private static final String _known_hosts="known_hosts";
+//  private static final String _known_hosts="known_hosts";
```

Comment:
If this isn't used, then I would say to remove it instead of commenting it out.

Reply:
I can delete it (as with the others) but my usual workflow is commenting out first and if nobody complains within some given time, I delete it in a later step.

Reply:
I think we can just safely remove it since it is a private field.